scene/gui/item_list.h

```diff
@@ -99,7 +99,7 @@ class ItemList : public Control {
 	SelectMode select_mode = SELECT_SINGLE;
 	IconMode icon_mode = ICON_MODE_LEFT;
 	VScrollBar *scroll_bar;
-	TextParagraph::OverrunBehavior text_overrun_behavior = TextParagraph::OVERRUN_NO_TRIMMING;
+	TextParagraph::OverrunBehavior text_overrun_behavior = TextParagraph::OVERRUN_TRIM_ELLIPSIS;
```

Comment:
I'm not entirely sure this is desired default mode for all case, but in general should be better the no trim.

Reply:
Run Godot with the `--doctool` command line argument to update the docs, see https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html for more info.